streams/src/test/java/org/apache/kafka/streams/state/NoOpWindowStore.java

```diff
@@ -61,4 +61,14 @@ public WindowStoreIterator fetch(final Object key, final long timeFrom, final lo
     public WindowStoreIterator<KeyValue> fetch(Object from, Object to, long timeFrom, long timeTo) {
         return null;
     }
+    
+    @Override
+    public WindowStoreIterator<KeyValue> all() {
```

Comment:
These should really return `KeyValueIterators.EMPTY_WINDOW_STORE_ITERATOR`

I realise the existing methods don't but they really should, too.

Reply:
@dguy I don't think that is possible because ```KeyValueIterators``` is part of the ```org.apache.kafka.streams.state.internals``` package. When I tried to import the class, I cannot seem to find it from ```NoOpWindowStore```. Not sure if that was supposed happen though.

Reply:
I've added the inner class of `NoOpWindowStore` when merging, and I think we could do it in a follow-up PR to make `KeyValueIterators` out of `internals` and remove this class.